tests/datasource/fluent/data_asset/data_connector/test_google_cloud_storage_data_connector.py

```diff
@@ -9,10 +9,10 @@
 from great_expectations.core import IDDict
 from great_expectations.core.batch import BatchDefinition
 from great_expectations.core.util import GCSUrl
+from great_expectations.datasource.fluent.batch_request import BatchRequest
```

Comment:
could the `__init__.py` be updated to avoid having to import BatchRequest from `batch_request`

Reply:
Yes! Right now we don't have any public API docs that show importing this class, so I opted to keep it narrowly namespaced at first. I will update the `__init__`.

Reply:
✅